ts/Series/Organization/OrganizationSeriesDefaults.ts

```diff
@@ -424,6 +426,52 @@ const OrganizationSeriesDefaults: OrganizationSeriesOptions = {
  * @apioption series.organization.nodes.image
  */
 
+/**
+ * The format string specifying what to show for *links* in the
+ * organization chart.
+ *
+ * Best to use with `linkTextPath` enabled.
+ *
+ * Defaults to `undefined`.
+ *
+ * @sample highcharts/series-organization/link-labels
+ *         Organization chart with link labels
+ *
+ * @type      {string}
+ * @product   highcharts
+ * @apioption series.organization.dataLabels.linkFormat
+ * @since 10.3.3
```

Comment:
if we are adding the option now, it will be available from the next release.
```suggestion
 * @since next
```